website/docs/reference/resource-properties/constraints.md

```diff
@@ -303,6 +303,67 @@ select
 
 </File>
 
+### Custom constraints on models for advanced configuration of tables
+
+In dbt Cloud, you can use custom constraints on models for advanced configuration of tables. Custom constraints allow you to set [masking policies](https://docs.snowflake.com/en/user-guide/security-column-intro#what-are-masking-policies) in Snowflake when using a  Create Table As Select (CTAS).
+
+Masking policies conceal sensitive data from unauthorized access, while still allowing authorized users to retrieve the data during query execution.
```

Comment:
I don't think that we should explain masking policies in this section of the docs. If people want to set them I'd expect them to know what they are.